### ISSUE TITLE
Quarto not detected with conda when using the rendering shortcut

### ISSUE THREAD

**Issue by cyrilcros** [quarto-dev/quarto#213]
Hi, I installed the `r-quarto` package in a Conda environment. I can activate and successfully render the documents when I activate the terminal. I used `which quarto` to find the path of quarto in this environment, and then set `quarto.path` in my `settings.json` to the same location with an absolute path, and reloaded the window.
When I try to render with "Ctrl-Shift-K" in VScode, I still receive a "please install the Quarto CLI" message.

**Comment by jjallaire** [comment]
In this case since the extension can't find Quarto on the PATH you'll need to use the `quarto.path` setting to specify exactly where Quarto is. You can do that or you can launch VS Code from a terminal _after_ you've activated the Conda environment.

**Comment by niowniow** [comment]
Changing the path is what OP did and also what I just tried. It does not work. I changed the `quarto.path`  to the conda install location, the message still appears. I'm using VSCode with a remote ssh connection and change the settings on the remote. afaik I cannot start vscode from my activated conda env in SSH mode.
It might be an issue that the vscode extension does not respect the conda environment to which the executable is installed. For example I cannot run the quarto executable outside of the conda environment. 
```
/miniconda3/envs/quartoenv/bin/quarto
/miniconda3/envs/quartoenv/bin/quarto: line 158: /miniconda3/envs/quartoenv/bin/tools/deno-x86_64-unknown-linux-gnu/deno: No such file or directory
```

And from the code it seems the Quarto VSCode extension tries to run the executable to test if quarto is installed, right?

**Comment by laguill** [comment]
Hello did you find a solution to use quarto executable from a conda environment in vscode ?

**Comment by maurosilber** [comment]
If it helps anyone, a (workaround) solution is to launch VSCode from the activated environment.

I tried it with `pixi`, and it works:
```
pixi shell
code .
```

But should be the same for `conda`:
```
conda activate
code .
```

**Comment by cyrilcros** [comment]
@maurosilber Yes, I realized at some point it is even mentioned at the bottom of https://quarto.org/docs/projects/virtual-environments.html#vs-code I personally just don't install quarto in a conda environment now because of this. In my specific case I was trying to do this via RemoteSSH on a SLURM cluster and it was complicated to start VScode from conda.

- My cluster has quarto installed as an Easybuild module, I can use `quarto.path` to that binary.
- Alternatively, there should be the option of using [VScode tasks](https://code.visualstudio.com/docs/debugtest/tasks) to launch Quarto, you could play with that
